.github/workflows/appsignals-e2e-eks-test.yml

```diff
@@ -258,6 +258,12 @@ jobs:
           curl -S -s http://${{ env.APP_ENDPOINT }}/remote-service?ip=${{ env.REMOTE_SERVICE_POD_IP }}/
           curl -S -s http://${{ env.APP_ENDPOINT }}/client-call/
 
+      - name: Initiate Gradlew Daemon
+        uses: ./.github/workflows/actions/execute_and_retry
+        with:
+          command: "./gradlew"
+          cleanup: "./gradlew clean"
```

Comment:
We can set the number of retries here higher

Reply:
Also, it doesn't look like the gradle is picking up the fact that the daemon is there [link](https://github.com/aws-observability/aws-application-signals-test-framework/actions/runs/8727081926/job/23943790575#step:13:82). Is building gradle in advance still helping somehow?

Reply:
Yeah building gradle in advance lets us reuse the cache so that we don't have to download it again. 
Gradle is picking up that the daemon is there, it's just shutdown so we have to start a new one